### PR TITLE
Remove poor device connectivity experiment

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

We collected enough data for this experiment.

Started in https://github.com/guardian/frontend/pull/25930

## Does this change need to be reproduced in dotcom-rendering ?

- [X] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## What is the value of this and can you measure success?

Results will be provided by D&I.

### CODE REVIEW

File: common/app/experiments/Experiments.scala [hunk 1]
```diff
@@ -42,7 +41,7 @@ object OfferHttp3
       name = "offer-http3",
       description = "Offer HTTP3 by providing the header and redirecting URLs to enable loading of assets with HTTP3",
       owners = Seq(Owner.withGithub("paulmr")),
-      sellByDate = LocalDate.of(2023, 4, 28),
+      sellByDate = LocalDate.of(2023, 3, 31),
```
Comment (OllysCoding) [inline common/app/experiments/Experiments.scala:44]: Why's the date of this one changed?

**Comment by ioannakok** [comment]
I think there are a few places where we check whether we're participating in this experiment: https://github.com/guardian/frontend/blob/ca5d77f3d7428bba4fddabf882ffa7e3db8abb02/common/app/views/fragments/page/head/stylesheets/styles.scala.html#L64

File: common/app/experiments/Experiments.scala (hunk 1 above)
Comment (mxdvl) [reply common/app/experiments/Experiments.scala]: No idea! Reverting.

**Review by ioannakok — APPROVED** [review]

**Comment by sndrs** [comment]
there is still code in DCR that responds to this too

**Pull request merged**